shopinvader_sale_profile/components/shopinvader_partner_export_mapper.py

```diff
@@ -11,5 +11,7 @@ class ShopinvaderPartnerExportMapper(Component):
     _inherit = "shopinvader.partner.export.mapper"
 
     @mapping
-    def sale_profile(self, record):
+    def role(self, record):
+        if not record.sale_profile_id.code:
```

Comment:
@rousseldenis The check should be on the backend configuration... `if not record.backend.use_sale_profile`